packages/flutter_tools/templates/app/pubspec.yaml.tmpl

```diff
@@ -3,7 +3,6 @@ description: {{description}}
 {{#withPluginHook}}
 publish_to: 'none'
 {{/withPluginHook}}
-{{^withPluginHook}}
```

Comment:
This will need a test case that verifies that the create example app has a version. There should be some existing examples in the test/ directory under create_test.dart

Reply:
Sounds good, will look into the test case